web/packages/teleterm/electron-builder-config.js

```diff
@@ -29,6 +30,10 @@ if (!isMac && env.CONNECT_TSH_BIN_PATH === undefined) {
   throw new Error('You must provide CONNECT_TSH_BIN_PATH');
 }
 
+if (isWindows && env.CONNECT_WINTUN_DLL_PATH == undefined) {
+  throw new Error('You must provide CONNECT_WINTUN_DLL_PATH');
```

Comment:
Could you expand `teleterm/README.md` with instructions on how to get wintun? Sometimes we build the packaged app locally.
You can add a new section for Windows under `## Build process`.